assets/src/js/resistance-hub-campaign.js

```diff
@@ -1,5 +1,21 @@
 const {__} = wp.i18n;
 
+// Re-order completed posts in a actions list block into resistance hub.
+const reorderCompletedPosts = () => {
+  const posts = document.querySelectorAll('.actions-list .wp-block-post-template > li');
```

Comment:
Actually I think you need to iterate it, like so `[...document.querySelectorAll('.actions-list .wp-block-post-template > li');]` because right now there is an error